x/fundraising/keeper/bid.go

```diff
@@ -112,71 +114,72 @@ func (k Keeper) PlaceBid(ctx sdk.Context, msg *types.MsgPlaceBid) (types.Bid, er
 	return bid, nil
 }
 
+// ValidateFixedPriceBid validates a fixed price bid type.
 func (k Keeper) ValidateFixedPriceBid(ctx sdk.Context, auction types.AuctionI, bid types.Bid) error {
-	if bid.Coin.Denom != auction.GetPayingCoinDenom() && bid.Coin.Denom != auction.GetSellingCoin().Denom {
+	if bid.Coin.Denom != auction.GetPayingCoinDenom() &&
+		bid.Coin.Denom != auction.GetSellingCoin().Denom {
 		return types.ErrIncorrectCoinDenom
 	}
 
 	if !bid.Price.Equal(auction.GetStartPrice()) {
-		return sdkerrors.Wrap(types.ErrInvalidStartPrice, "start price must be equal to the auction start price")
+		return sdkerrors.Wrap(types.ErrInvalidStartPrice, "start price must be equal to the start price of the auction")
 	}
 
-	// PayingCoinAmount / Price = BidSellingCoinAmount
-	bidSellingAmt := bid.GetBidSellingAmount(auction.GetPayingCoinDenom())
-	bidSellingCoin := sdk.NewCoin(auction.GetSellingCoin().Denom, bidSellingAmt)
+	// For remaining coin validation, convert bid amount in selling coin denom
+	bidAmt := bid.ConvertToSellingAmount(auction.GetPayingCoinDenom())
+	bidCoin := sdk.NewCoin(auction.GetSellingCoin().Denom, bidAmt)
+	remainingCoin := auction.GetRemainingSellingCoin()
 
-	// The bidder can't bid more than the remaining selling coin
-	remainingSellingCoin := auction.GetRemainingSellingCoin()
-	if remainingSellingCoin.IsLT(bidSellingCoin) {
-		return sdkerrors.Wrapf(types.ErrInsufficientRemainingAmount, "remaining selling coin amount %s", remainingSellingCoin)
+	if remainingCoin.IsLT(bidCoin) {
+		return sdkerrors.Wrapf(types.ErrInsufficientRemainingAmount, "remaining selling coin amount %s", remainingCoin)
 	}
 
-	// Get the total bid amount by the bidder
+	// Get the total bid amount bid by the bidder
```

Comment:
```suggestion
	// Get the total bid amount bid by the bidder
```

It seems `bid` is used once more like:
```suggestion
	// Get the total bid amount by the bidder
```

Reply:
I will update with the suggested change. Thanks!